identity/pom.xml

```diff
@@ -19,6 +19,53 @@
     <maven.compiler.source>21</maven.compiler.source>
     <maven.compiler.target>21</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <spring-boot.version>3.2.5</spring-boot.version>
```

Comment:
:wrench: these are decalred in the `/parent` module and provide a centralised version management, I'd use the versions there and add any that we need to that